spec/admin.mediawiki

```diff
@@ -49,23 +49,23 @@ The server will provide an HTTP API for performing various adminstrative tasks.
 
 '''API Endpoints'''
 {|
-! path      !! description
+! path      !! type !! description
```

Comment:
`type` is pretty clear when you look at the table, but `method` is more precise as these are http requests methods.